applications/openshift/high-availability/worker_nodes_in_two_zones_or_more/tests/ocp4/e2e.yml

```diff
@@ -0,0 +1,2 @@
+---
+default_result: FAIL
```

Comment:
My current cluster and the cluster CI ran on have two zones.
Not sure if this is always the case. We can add a comment and set this to PASS for now.

Comment can be something like:
`# This rule may fail depending on how the Cluster is deployed`